server/src/main/resources/db/migration/V32__store_nlps.sql

```diff
@@ -1,8 +1,8 @@
 INSERT INTO `natural_text_actions` (`id`, `workspace_type`, `natural_text`, `data`, `display_name`, `snippet_class`, `action`,  `created_date`, `updated_date`,`condition_type`, `allowed_values`,  `import_to_web`, `import_to_mobile_web`, `import_to_android_native`, `import_to_ios_native`) VALUES
```

Comment:
@creek23 You need to add update statements in a new migration file instead of changing the old migration files

Reply:
I'm preparing the SQL file with UPDATE statements which I believe can be used by existing install but I'd like to ask: Why not correct this file now -- why should we allow fresh installs to have this mistake?

Reply:
There are 2 scenarios here. For fresh install, your suggestion makes sense. But for migration from previous versions to the latest, the checksum for the migration files will fail. The migration files once applied will be stored in a db table with its calculated checksum. Once you change the file, the checksum wont match

Reply:
I don't know how Testsigma Server handles migration; for the "new migration file" you are asking, what filename should it be? -- I mean, how can Testsigma Server know that a new migration file is waiting to be triggered?  Since it's modifying 2 separate tables, should it be 2 separate SQL files?

Reply:
@creek23 The filename can be anything as long as it follows the existing naming pattern. Testsigma tracks all the migration files it has applied till now in the table `flyway_schema_history`